src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -44,11 +44,14 @@ namespace Google.Example.Library.V1.Samples
             // Shelf deleted
             Console.WriteLine("Shelf deleted.");
         }
-        // [END sample]
+    }
+
+    // [END sample]
 
+    public class DeleteShelfAsyncFlattenedAsyncEmptyResponseTypeWithResponseHandlingMain {
```

Comment:
The naming is a little crazy!

Reply:
Yes! That's to test all samples have unique class names when users happen to provide the same name for two samples. It should never happen in production :)